src/Modera/BackendSecurityBundle/Resources/public/js/toolscontribution/view/user/EditWindow.js

```diff
@@ -40,6 +41,11 @@ Ext.define('Modera.backend.security.toolscontribution.view.user.EditWindow', {
                         name: 'id'
                     },
```

Comment:
Is there any specific reason why you woudln't just make this field read-only (disabled), or label ? Is is that Ext From doesn't return values for disabled fields ?